main.js

```diff
@@ -15,6 +15,14 @@ const popUp = document.querySelector(".desk-popup");
 const closeButton2 = document.querySelector(".close-popup");
 const professionalArtDiv = document.querySelector(".professional-art");
 
+const error = document.getElementById("error");
+const submitForm = document.getElementById("form");
+const submitBtn = document.getElementById("btn__submit");
```

Comment:
- [ ] After filling the form, and not submitting it, the entered values should be saved in the browser. but I noticed that nothing has been saved after filling the form :upside_down_face:  kindly fis this issue

Reply:
- [ ] This particular requirement has not been implemented. Kindly look into it and fix it.


https://user-images.githubusercontent.com/26597432/202730592-8e31a86a-14e3-40ab-b39c-5181001adda9.mp4